CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contribution Guidelines
+
+Thanks for your interest in contributing to PyCon India Blog. Your valuable contribution is much appreciated.
+
+The following is a set of guidelines for contributing to PyCon India Blog.
+
+## How Can I Help?
+
+Start looking through the issues and watch out for any issues with the `bug` or `help wanted` labels. Issues marked with these labels haven't been assigned to anyone yet and we would love for you to work on them.
+
+## Request a feature
+
+If you think there is a cool feature you would like to add please create an issue for that and we will assign that to you if the feature looks good to add.
```

Comment:
It's not mandatory that the user who has created the issue has to work on this. It would be better this way, `If you think there is a cool feature you would like to add please create an issue for that. You can also mention if you would like to work on this issue and it can be assigned to you by moderators.`

Reply:
updated